.github/workflows/new-windows-post-2019-2-editor-image-requested.yml

```diff
@@ -7,25 +7,25 @@ on:
   workflow_dispatch:
     inputs:
       jobId:
-        description: 'Job ID'
+        description: "Job ID"
         required: true
-        default: 'dryRun'
+        default: "dryRun"
       editorVersion:
-        description: 'Unity Editor Version'
+        description: "Unity Editor Version"
         required: true
-        default: '2020.3.24f1'
+        default: "2020.3.24f1"
```

Comment:
Last time I checked yaml usually uses single quotes. This includes all GHA examples. 

Unless this is done by our linter rules I suggest we also stick to single quotes instead.

Reply:
This seems to have been done by the GitHub actions vscode plugin as when I saved in vscode it auto replaced them all. Since it's by GitHub themselves I figured it would be fine 🤷‍♂️.